site-book/README.md

```diff
@@ -0,0 +1,51 @@
+# Site Book Metron documentation 
+
+Metron's Site Book is an attempt at producing documentation that is
+
+- Versioned and reviewed,
+- Only committers can edit
+- Tied to code versions
+- Highly local to the code being documented
+
+Currently it is in a stand-alone, versioned-with-the-code sub-directory and sub-project. The idea is that a release
+manager would build the site-book (following the instructions below), then copy it into a versioned subdirectory of the (unversioned) public site, to publish it along with each code release.
```

Comment:
suggest instead this sentence to read:
"The idea is that a release manager would build the site-book (following the instructions below), then publish it from the public Metron site (http://metron.incubator.apache.org/) as the docs for the new released version.  Older site-book versions should remain available for users that need them."